styles.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Wrote this just to see if our white svg images were popping up on the previously white background, and we just couldn't see them. The svg images were not there, so this is no longer necessary. Let's trash it!